app/style/Header.less

```diff
@@ -1,123 +1,114 @@
-  background-size: 30px;
-  background-repeat: no-repeat;
-}
-
-.header-icon-transactions {
-  .header-icon-mixin;
-
-  background-image: @transactions-big-icon;
-}
-
-.tabbedheader-title {
-  height: 54px;
-  line-height: 49px;
-  color: #596d81;
-  font-size: 27px;
-  text-transform: capitalize;
-  //padding-left: 50px;
-}
-
-.tabbedheader-description {
-  height: 54px;
-  width: 480px;
-  padding-left: 50px;
-  color: #0c1e3e;
-  font-size: 14px;
-}
-
-.tabbedheader-icon::before {
-  content: "";
-}
-
-.tabbedheader-icon {
-  padding-left: 30px;
-  margin-right: 20px;
-}
-
-.tabbedheader-tab {
-  display: inline-block;
-  vertical-align: middle;
-  text-align: center;
-  padding: 0px 4px 3px 4px;
-  margin: 0px 10px 0px 10px;
-  box-sizing: border-box;
-  color: inherit;
-  text-decoration: none;
-}
-
-.tabbedheader-tabs {
-  position: relative;
-}
-
-.tabbedheader-tab.active {
-  border-bottom: 5px solid #2ed8a3;
-}
-
-.tabbedheader-active-tab-caret {
-  height: 5px;
-  width: 15px;
-  background-color: #2ed8a3;
-  position: relative;
-}
-
-.tabbedheader-active-tab-caret.updating {
-  transition: width 130ms @easeInOutQuart;
-  transition: left 210ms @easeInOutQuart;
-}
+@import (reference) "./main.less";
+@import (reference) "./Icons.less";
+
+.header-get-started {
+  padding-right: 42px;
+  padding-left: 80px;
+  background-color: #596d81;
+}
+
+.header-top-get-started {
+  height: 34px;
+  padding-top: 5px;
+  text-align: center;
+}
+
+.header-title-overview-get-started {
+  height: 54px;
+  padding-top: 13px;
+  color: #fff;
+  font-size: 27px;
+}
+
+.header-meta-overview-get-started {
+  height: 54px;
+  padding-top: 5px;
+  font-size: 18px;
+  color: #c4cbd2;
+  font-family: Inconsolata, monospace;
+}
+
+.header {
+  padding-right: 42px;
+  padding-left: 80px;
+  background-color: #fff;
+}
+
+.header-top {
+  height: 34px;
+  padding-top: 5px;
+  text-align: center;
+}
+
+.header-title-overview {
+  height: 54px;
+  padding-top: 13px;
+  color: #596d81;
+  font-size: 27px;
+}
+
+.header-meta-overview {
+  height: 54px;
+  padding-top: 5px;
+  font-size: 53px;
+}
+
+.header-icon-mixin {
+  background-position: 0px 50%;
+  background-size: 30px;
+  background-repeat: no-repeat;
+}
+
+.header-icon-transactions {
+  .header-icon-mixin;
+
+  background-image: @transactions-big-icon;
+}
+
+.tabbedheader-title {
+  height: 54px;
+  line-height: 49px;
+  color: #596d81;
+  font-size: 27px;
+  text-transform: capitalize;
+  //padding-left: 50px;
+}
+
+.tabbedheader-description {
+  height: 54px;
+  width: 480px;
+  padding-left: 50px;
+  color: #0c1e3e;
+  font-size: 14px;
+}
+
+.tabbedheader-icon::before {
+  content: "";
+}
+
+.tabbedheader-icon {
+  padding-left: 30px;
+  margin-right: 20px;
+}
+
+.tabbedheader-tab {
+  display: inline-block;
+  vertical-align: middle;
+  text-align: center;
+  padding: 0px 4px 3px 4px;
+  margin: 0px 10px 0px 10px;
+  box-sizing: border-box;
+  color: inherit;
+  text-decoration: none;
```

Comment:
I see the text-decoration set to none, but looks like the tabs have default `<a>` styling.

Reply:
this is from before... I didn't add or change any styling rules.

Reply:
Well no matter the cause, it needs to be fixed